app/script/calling/CallingRepository.js

```diff
@@ -1031,6 +926,12 @@ z.calling.CallingRepository = class CallingRepository {
         amplify.publish(z.event.WebApp.WARNING.SHOW, z.viewModel.WarningsViewModel.TYPE.UNSUPPORTED_OUTGOING_CALL);
         throw new z.calling.CallError(z.calling.CallError.TYPE.NOT_SUPPORTED);
       }
+
+      const isVideoCall = mediaType === z.media.MediaType.AUDIO_VIDEO;
```

Comment:
Maybe we should check all the possible type that can be considered a video.

- AUDIO_VIDEO
- VIDEO
- SCREEN